test/screenshotter/ss_data.yaml

```diff
@@ -0,0 +1,85 @@
+Accents: \vec{A}\vec{x}\vec x^2\vec{x}_2^2\vec{A}^2\vec{xA}^2
```

Comment:
No, that was manually converted, with lots of search & replace, some macros and manual line wrapping. Will add comments.